content/millennium/dstu2/conformance/conformance.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Conformance | DSTU 2 API
+---
+
+# Conformance
+
+* TOC
+{:toc}
+
+## Overview
+
+The Conformance resource documents the functionality implemented from the HL7<sup>®</sup> FHIR<sup>®</sup> standard. Unlike most resources,
+this resource is found at [`:serviceRootURL/metadata`] instead of by its resource name.
+
+## Metadata
+
+Get the capabilities and configurations of this implementation and deployment of the FHIR standard.
+
+    GET /metadata?:parameters
+
+_Implementation Notes_
+
+* Authentication is not required to access the Conformance resource
+* No parameters other than the standard `_format` parameter are supported for reading Conformance
+
+### Authorization Types
+
+Authorization is not required
```

Comment:
add a period

Reply:
Made the change here: [Fix](https://github.com/cerner/fhir.cerner.com/pull/323/commits/7a5c7155c88812a895674b0212f9b08f67a89fa7#diff-0348b9d7d711f1f3dae95b0f72a7633cL28)